node/Cargo.toml

```diff
@@ -24,71 +24,73 @@ hex-literal = "0.3.4"
 log = "0.4"
 serde = { version = "1.0.144", features = ["derive"] }
 serde_json = "1.0"
-tracing-core = "=0.1.26" # fixate the version, see https://github.com/paritytech/substrate/issues/11691
+# Conflict of deps from <cumulus-client-collator "0.1.25"> and <cumulus-client-consensus-aura "0.1.36">
+tracing = "0.1.36"
```

Comment:
I don't think we need this. Does it compile without this line?

Reply:
It needs this line to suppress other deps. As the comment pointed out. Otherwise there will be deps conflict.

Reply:
double checked. you are right.